lib/plugins/transform-dot-component-invocation.js

```diff
@@ -0,0 +1,85 @@
+
```

Comment:
Can you link to this same file in emberjs/ember.js and explain why we have to include it here? 

Also, can you remove some of these docs (I think a single example is probably enough)?